dashboard/app/models/pd/workshop.rb

```diff
@@ -634,6 +644,14 @@ def fit_weekend?
     ].include?(subject)
   end
 
+  def funded_csf?
+    course == COURSE_CSF && funded
+  end
+
+  def funded_friendly_name
```

Comment:
nit: This isn't really a name, it's a string. Funded_friendly_string? Funded_string?

Reply:
How about `funding_summary`?